### ISSUE TITLE
`reaction-avatars`:   null is not an object (evaluating 'commentReactions.closest("[data-body-version]").dataset')

### ISSUE THREAD

**Issue by fregante** [sindresorhus/refined-github#4845]
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.

### Description

I see this error in the console

### Console errors

```shell
Unhandled Promise Rejection: TypeError: null is not an object (evaluating 'commentReactions.closest("[data-body-version]").dataset')
```

### Example URL

https://togithub.com/pixiebrix/pixiebrix-extension/pull/1548

### Browser(s) used

safari

**Comment by cheap-glitch** [comment]
This happens whenever reacting on a _review comment_, because they're not wrapped in a div with the `data-body-version` attribute. Also, only the reactions part gets replaced instead of the whole comment wrapper.

https://github.com/sindresorhus/refined-github/blob/1d858670461a256b29dda730112a0e965b48f6f2/source/features/reactions-avatars.tsx#L70-L72

Is there a reason why `onReplacedElement()` was used here instead of `onElementRemoval()` directly? Otherwise, this is a pretty easy fix (and solves #4591 as an added bonus):
```ts
await onElementRemoval(commentReactions.closest('.comment-reactions')!);
init();
```

**Comment by fregante** [comment]
Not sure, but `onReplacedElement` is supposed to be the higher level version, basically a `selector-observer` for existing elements that are just updated over time.
